src/mobile/apps/home/templates/page.jade

```diff
@@ -18,7 +18,12 @@ block content
                     h2!= heroUnit.subtitle
 
   section.main-side-margin.home-page-section
-    h1.avant-garde-header-center Explore Artsy
+    if collectionsHubs && collectionsHubs.length
+      h2 Featured categories
+      != stitch.components.CollectionsHubsHomepageNav({collectionsHubs: collectionsHubs})
+    else
+      h1.avant-garde-header-center Explore Artsy
```

Comment:
Are the `h2` and the `h1` meant to be mutually exclusive? (Seems like if hubs are turned on we lose the Explore Artsy header here)

Reply:
Yes - according to the Zeplin specs, the "Explore Artsy" header isn't there when the hubs are showing.